ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1552,6 +1764,97 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    // For Grand Octet:
+    // After bosses and dragons start spawning, there's no clear log line we can trigger off of to find bosses' position
+    // until it's effectively too late.  The best way to do this seems to be to fire the trigger
+    // with a delay when Bahamut uses Grand Octet before all 3 bosses jump.
+    {
+      id: 'UCU Grand Octet Run & Rotate',
+      type: 'Ability',
+      // Grab mob position data after dragons/bosses are positioned
+      netRegex: { id: '26E7', source: 'Bahamut Prime', capture: false },
+      delaySeconds: 4.8,
```

Comment:
Where did this delay come from?

Reply:
This is the amount of time it takes after Bahamut Prime uses Grand Octet (`26E7`) for the trio to appear mid, disappear, and for Bahamut and Nael to then respawn on the perimeter (which gives the position data we need to calculate both the direction to run and rotation - plus a little safety margin for potential network lag.  Tested extensively during prog and I (and others) found this threshold to be reliable.